packages/pigeon/lib/java_generator.dart

```diff
@@ -237,15 +243,15 @@ String _castObject(Field field, List<Class> classes, String varName) {
 
 /// Generates the ".java" file for the AST represented by [root] to [sink] with the
 /// provided [options].
-void generateJava(JavaOptions options, Root root, StringSink sink) {
+void generateJava(JavaOptions? options, Root root, StringSink sink) {
```

Comment:
The options should be non-null, className should be non-null, package should be nullable  =)

Reply:
Alright, `className` is now required.